changes/ce/fix-12530.en.md

```diff
@@ -0,0 +1 @@
+Enhanced `frame_too_large` and malformed CONNECT packet parse failures to include more information to help troubleshooting.
```

Comment:
```suggestion
Enhanced `frame_too_large` and malformed CONNECT packet error logs to include more information to help troubleshooting.
```

Otherwise it may sound like the PR has improved _failures_.

Reply:
thanks. will fix in the next commit to come